File: JavaScript/largest-element-in-an-array.js
class Solution {
    largest(arr, n) {
        let max = Number.NEGATIVE_INFINITY;

        for (let i = 0; i < arr.length; i++) {
            if (arr[i] > max) {
                max = arr[i];
            }
        }

        return max;
    }
}

File: C++/largest-element-in-an-array.cpp
// User function Template for C++

class Solution {
public:
    int largest(int arr[], int n) {
        int maxVal = arr[0];   // start with the first element

        for (int i = 1; i < n; i++) {
            if (arr[i] > maxVal) {
                maxVal = arr[i];
            }
        }

        return maxVal;
    }
};
